src/tool/yaml2sql.cpp

```diff
@@ -190,7 +191,24 @@ bool process( const std::string& type, uint32 version, const std::vector<std::st
 	return true;
 }
 
+/**
+ * process the arguments
+ * @retval 0 = continue program
+ * @retval 1 = exit program (like --help)
+ */
+int process_args(int argc, char ** argv) {
```

Comment:
```suggestion
bool process_args(int argc, char ** argv) {
```
Technically we should only be returning on certain cases so we really will only have a "pass" or "fail" return option.